src/openrct2/actions/RideSetAppearanceAction.cpp

```diff
@@ -139,7 +139,7 @@ GameActions::Result::Ptr RideSetAppearanceAction::Execute() const
             gfx_invalidate_screen();
             break;
     }
-    window_invalidate_by_number(WC_RIDE, _rideIndex);
+    window_invalidate_by_number(WC_RIDE, static_cast<int32_t>(_rideIndex));
```

Comment:
This is used so much, I think we should create a function for it. E.g. `WindowRideInvalidateById(ride_id_t id)`. It would make the code much cleaner.

Reply:
I agree, theres a lot of UI code that needs some cleaning.